webpack.config.js

```diff
@@ -0,0 +1,57 @@
+const path = require('path');
+const TerserPlugin = require('terser-webpack-plugin');
+
+const nodeEnv = process.env.NODE_ENV || 'development';
+const isProd = (nodeEnv === 'production');
+
+module.exports = {
+  mode: nodeEnv,
+  optimization: {
+    minimize: isProd,
+    minimizer: [
+      new TerserPlugin({
+        terserOptions: {
+          compress:{
+            drop_console: true,
+          }
+        }
+      }),
+    ],
+  },
+  entry: {
+    dist: './h5p-editor-uuid.ts'
+  },
+  output: {
+    filename: 'h5p-editor-uuid.js',
+    path: path.resolve(__dirname, 'dist')
```

Comment:
It's helpful to also have `clean: true` in order to always clean up the distribution folder. Some unrequired stuff might end up in there if one develops with in `watch` mode with `development` set and then packs after building the `production` files.